### ISSUE TITLE
Variant property should not be in DocumentResponse type

### ISSUE THREAD

**Issue by ricfrank** [onfido/onfido-sdk-ui#1472]
### What was the expected behaviour?
When `onComplete` callback is called, in the `DocumentResponse` type should not be `variant` property.

### What happened instead?
When `onComplete`callback is called, I found `variant` property in the `SdkResponse.DocumentResponse`.

### Version info:

- Onfido SDK 6.9.0
- OS MacOs Catalina 10.15.7
- Browser Firefox 89.0
- Device
- Node 12.9.1
- Framework or relevant dependencies (if applicable)

### Integration configuration:

```
initOnfido({
                useModal: false,
                token: applicant.token,
                language: {
                    locale: 'en_US',
                    phrases,
                },
                smsNumberCountryCode: 'IT',
                onComplete: (data: OnfidoOnCompleteData) => {
                    onComplete(data, applicant, borrower);
                },
                onError: this.onError,
                customUI: {
                    colorBackgroundButtonPrimary: soisyGreenColor,
                    colorBorderButtonPrimary: soisyGreenColor,
                    colorBackgroundButtonPrimaryActive: soisyGreenColor,
                    colorBackgroundButtonPrimaryHover: '#77c989',

                    colorBorderButtonSecondary: soisyGreenColor,
                    colorContentButtonSecondaryText: soisyGreenColor,

                    colorBackgroundInfoPill: soisyGreenColor,

                    colorBackgroundLinkHover: soisyGreenColor,
                    colorBorderLinkUnderline: soisyGreenColor,
                    colorBackgroundLinkActive: soisyGreenColor,
                },
                steps: [
                    {
                        type: 'document',
                        options: {
                      
                            documentTypes: selectDocumentType(borrower.documentType),
                            forceCrossDevice: true,
                            showCountrySelection: false,
                        },
                    },
                    {
                        type: 'face',
                        options: {
                            requestedVariant: 'video',
                        },
                    },
                ],
            });
        };
```

### Steps to reproduce:

In `data` parameter of `onComplete` callback, there is the `.variant` proprerty that there shouldn't be. 

### Demo:

<img width="551" alt="Schermata 2021-06-09 alle 20 59 04" src="https://user-images.githubusercontent.com/892169/121413142-b0cc7000-c965-11eb-9bc0-c5d762571f50.png">

**Comment by ethan605** [comment]
Hi @ricfrank, thanks for raising this. The `variant` property was missing in the [type declaration of `DocumentResponse`](https://github.com/onfido/onfido-sdk-ui/blob/fc275dc319a1ed0a0b8b092d4b8ece1610477609/src/types/sdk.ts#L6-L10). We'll fix this in the next release of Web SDK.

**Comment by ricfrank** [comment]
Thanks for fixing. 

Why ```DocumentResponse``` type needs to ```variant``` property ?

**Comment by ethan605** [comment]
Hi @ricfrank, there are 2 reasons:
1. To normalise the fields between `DocumentResponse` and `FaceResponse`.
2. We're planning to bring more features that involve with `requestedVariant` of `document` step.

**Comment by kopijunkie** [comment]
This fix has been released in `6.11.1` so I'll close this ticket now.
